File: Modul 5/C/PRAK504-2310817220025-FirdaKhoirunisa.c
#include <stdio.h>

int reverse(int angka){
    int reverse = 0;
    int remainder;
    while (angka != 0){
        remainder = angka % 10;
        reverse = reverse * 10;
        reverse = reverse + remainder;
        angka = angka / 10;
    }return reverse;
}

int main(){
    int A, B;
    scanf("%d %d", &A, &B);
    A = reverse(A);
    B = reverse(B);
    int C = A + B;
    printf("%d", reverse(C));
}

File: Modul 5/Python/PRAK504-2310817220025-FirdaKhoirunisa.py
def reverse (angka) :
    reversed = 0
    while int(angka) != 0:
        remainder = int(angka) % 10
        reversed = reversed * 10 + remainder
        angka = int(angka) / 10
    return reversed

A, B = input().split()
A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))